packages/client-core/src/admin/components/Setting/Authentication.tsx

```diff
@@ -461,33 +461,6 @@ const Account = () => {
         </Grid>
 
         <Grid item xs={12} sm={6} md={6}>
-          <Typography className={styles.settingsSubHeading}>{t('admin:components.setting.local')}</Typography>
-
-          <InputText
-            name="username"
-            label={t('admin:components.setting.userName')}
-            value={authSetting?.local.usernameField || ''}
-            disabled
-          />
-
-          <InputText
-            name="password"
-            label={t('admin:components.setting.password')}
-            value={authSetting?.local.passwordField || ''}
```

Comment:
We can also remove `local` section from line 221 in `/server-core/src/appconfig.ts` & line 22 in `packages/server-core/src/setting/authentication-setting/authentication.seed.ts`

![image](https://user-images.githubusercontent.com/10975502/229702368-a781e1a1-2740-4d37-baf5-b0fa6d072b25.png)